src/bin/encode.rs

```diff
@@ -92,12 +92,10 @@ fn handle_path<'a>(options: &mut Options<'a>, source_path: &Path, sub_dir: &Path
                 .expect("Could not find or create destination directory");
 
             let mut bin_path = d.join(sub_dir);
-            bin_path.push(file_name);
-            bin_path.set_extension("binjs");
+            bin_path.push(file_name.to_string_lossy().to_string() + ".binjs");
```

Comment:
It might be better to do `format!("{}.binjs", file_name.to_str().unwrap())` - aside from efficiency differences, this will allow to fail on cases when filename contains unsupported characters rather than silently replace them.

In the future we might want to support them properly, but it's unlikely to be a common case, so just failing would be good for now.